segmentation/utils/spatial_analysis.py

```diff
@@ -6,43 +6,42 @@
 importlib.reload(spatial_analysis_utils)
 
 
-def calculate_channel_spatial_enrichment(dist_matrices, marker_thresholds, all_data,
-                                         excluded_colnames=None, fovs=None,
-                                         dist_lim=100, bootstrap_num=1000):
+def calculate_channel_spatial_enrichment(dist_matrices_dict, marker_thresholds, all_fov_data,
```

Comment:
These are all good except for all_fov_data, I think all_data is simpler. FOV doesn't actually provide any information here